mithril-stm/src/multi_sig.rs

```diff
@@ -641,8 +641,31 @@ mod tests {
     use crate::key_reg::KeyReg;
     use proptest::prelude::*;
     use rand_chacha::ChaCha20Rng;
-    use rand_core::{OsRng, SeedableRng};
+    use rand_core::SeedableRng;
+
+    // ---------------------------------------------------------------------
+    // Test helpers
+    // ---------------------------------------------------------------------
+    impl PartialEq for SigningKey {
+        fn eq(&self, other: &Self) -> bool {
+            self.0.to_bytes() == other.0.to_bytes()
+        }
+    }
 
+    impl Eq for SigningKey {}
+
+    // ---------------------------------------------------------------------
+    // Property test: `test_sig`
+    // Property test: `test_invalid_sig`
+    // Property test: `test_infinity_sig`
+    // Property test: `test_infinity_vk`
+    // Property test: `test_keyreg_with_infinity_vk`
+    // Property test: `test_aggregate_sig`
+    // Property test: `test_eval_sanity_check`
+    // Property test: `serialize_deserialize_vk`
+    // Property test: `serialize_deserialize_sk`
+    // Property test: `batch_verify`
+    // ---------------------------------------------------------------------
```

Comment:
I'm not sure that all these comments before property tests are interesting after all as we can easily find them without them :thinking: